gene_benchmark/descriptor.py

```diff
@@ -4,6 +4,57 @@
 
 import mygene
 import pandas as pd
+import requests
+
+
+def _gene_symbol_to_ensemble_ids(
+    symbols: list[str], species: str = "human"
+) -> dict[str, str]:
+    """
+    converts between symbols to their ensemble ids.
+
+    Args:
+    ----
+        symbols (list[str]): A list of gene symbols
+        species (str, optional): The species for the ensemble id conversion. Defaults to "human".
+
+    Returns:
+    -------
+        dict[str, str]: dictionary with symbol ensemble pairs (if symbol didn't find a ensemble id it will not appear)
+
+    """
+    mg = mygene.MyGeneInfo()
+    query = " ".join(symbols)
+    gene_info = mg.query(query, fields="symbol,ensembl.gene", species=species)
+    ids = {}
+    for hit in gene_info["hits"]:
+        symbol = hit["symbol"]
+        ensembl_id = hit.get("ensembl", {}).get("gene", None)
+        ids[symbol] = ensembl_id
+    return ids
+
+
+def _fetch_ensembl_sequence(ensembl_gene_id):
+    """
+    retries the base pair sequence of a given ensemble id using ensembl.org API.
```

Comment:
**retrieves** the base pair sequence  ... ? returns none if not found. 
should we add this url as an input?

Reply:
I think note because the method can work with that URL only